lib/MusicBrainz/Server/Email.pm

```diff
@@ -624,6 +625,34 @@ EOF
 
     my $email = $self->_create_email(\@headers, $body);
     $self->_send_email($email);
+
+    if ($opts{send_to_self}) {
+        my $copy_subject = 'Copy of your report of ' . $reported_user_name;
+
+        my @copy_headers = (
+            'To'          => _user_address($opts{reporter}),
+            'Sender'      => $EMAIL_NOREPLY_ADDRESS,
+            'Subject'     => _encode_header($copy_subject),
+            'Message-Id'  => _message_id('editor-report-copy-%s-%s-%d', $reporter->id, $reported_user->id, time),
```

Comment:
Wasn't 100% sure what to set as message ID - I understand it should be different from the ID for the actual report, so I added `-copy-`. I assume `time` wouldn't be 100% the same so maybe it's ok to just have this be the same? (that's what we seem to do for the copy of submitted user-to-user messages)

Reply:
I think what you have should be fine.